src/main/java/org/folio/consortia/service/FolioTenantService.java

```diff
@@ -62,6 +68,37 @@ public FolioTenantService(JdbcTemplate jdbcTemplate,
     this.customFieldsRetryProperties = customFieldsRetryProperties;
   }
 
+  /*
+   * Because of the liquibase.Scope implementation for the SpringLiquibase it is not possible to run several
+   * SpringLiquibase executions simultaneously. That is why this method must be synchronized.
+   */
+  @Override
+  public synchronized void createOrUpdateTenant(TenantAttributes tenantAttributes) {
+    beforeTenantUpdate(tenantAttributes);
```

Comment:
Is it necessary to make this method call here?

Reply:
it is empty, also `beforeLiquibaseUpdate` and `afterLiquibaseUpdate`, thus I am gonna get rid of them